test/acceptance/integration/steps_lib/mock_utils.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+
+This file is part of fiware-orion-pep
+
+fiware-orion-pep is free software: you can redistribute it and/or
+modify it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the License,
+or (at your option) any later version.
+
+fiware-orion-pep is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with fiware-orion-pep.
+If not, seehttp://www.gnu.org/licenses/.
+
+For those usages not covered by the GNU Affero General Public License
+please contact with::[iot_support@tid.es]
+"""
+import requests
+from integration.tools.general_utils import start_mock, check_world_attribute_is_not_none
+
+__author__ = 'Jon'
+
+from lettuce import step, world
+
+# Mock start
+@step('a started mock')
+def start_mock_step(step):
+    world.mock_pid = start_mock()
+
+
+@step('retrieve information from the mock')
```

Comment:
Probably "the mock" is too generic and something more specific should be used along the files. E.g. "the context broker mock", "the notification receiver mock", "the context provider mock".

At least for me is a bit confusing... although I may be biassed :). Thus, if changing name makes no sense, mark this comment thread with NTC.